src/main/kotlin/io/S5p.kt

```diff
@@ -114,10 +140,21 @@ object S5p {
                     lyric = it.lyric,
                     pitch = it.key
                 )
-            }
+            },
+            parameters = emptyTrack.parameters!!.copy(
+                pitchDelta = generatePitchData(track, features)
+            )
         )
     }
 
+    private fun generatePitchData(track: model.Track, features: List<Feature>) : List<Double> {
+        if (!features.contains(Feature.CONVERT_PITCH)) return emptyList()
+        val data = track.pitch?.getRelativeData(track.notes)
+            ?.map { (it.first / 3.75) to (it.second * 100) }
```

Comment:
ditto. The `3.75` here should be calculated by TICK_RATE and `interval` passed from the the caller.